src/views/InProgress.vue

```diff
@@ -27,31 +27,29 @@
           </ion-label>
           <ion-icon :icon="printOutline" />
         </ion-item>
-      </div> 
-
+      </div>
       <ion-button expand="block" class="desktop-only" fill="outline" @click="packOrdersAlert">{{ $t("Pack orders") }}</ion-button>
-
-      <ion-card>
+      <ion-card v-for="(orders, index) in inProgressOrders.list.inProgress" :key="index">
```

Comment:
The data should be returned and stored in a state in a simple manner to be used directly and we don't need to do this much nesting.